Vungle/VungleRouter.m

```diff
@@ -301,24 +304,26 @@ - (void)requestBannerAdWithPlacementID:(NSString *)placementID
             [dictionary setObject:[NSNumber numberWithInt:BannerRouterDelegateStateRequesting] forKey:kVungleBannerDelegateStateKey];
             [self.bannerDelegates addObject:dictionary];
 
-            NSError *error = nil;
-            if (CGSizeEqualToSize(size, kVNGMRECSize)) {
-                if ([[VungleSDK sharedSDK] loadPlacementWithID:placementID error:&error]) {
-                    MPLogInfo(@"Vungle: Start to load an ad for Placement ID :%@", placementID);
-                } else {
-                    [self requestBannerAdFailedWithError:error
-                                             placementID:placementID
-                                                delegate:delegate];
-                }
-            } else {
-                if ([[VungleSDK sharedSDK] loadPlacementWithID:placementID withSize:[self getVungleBannerAdSizeType:size] error:&error]) {
-                    MPLogInfo(@"Vungle: Start to load an ad for Placement ID :%@", placementID);
-                } else {
-                    if ((error) && (error.code != VungleSDKResetPlacementForDifferentAdSize)) {
+            if (needRequestAd) {
```

Comment:
If `needRequestAd` is NO, do we need to add the delegate to the `bannerDelegates`. Thanks :)

Reply:
Yes. Since we already requested with `startWithAppId:appId options:initOptions error:&error` for this placement, we need `delegate` to get callback.

Reply:
@ashinagawa Got it, thanks :)